bedrock/base/templates/includes/structured-data/organizations/mozilla-corporation-organisation.json

```diff
@@ -0,0 +1,518 @@
+    "logo": "{{ settings.CANONICAL_URL + static('img/logos/firefox/quantum/logo-lg-high-res.png') }}",
+    "image": [
+      "{{ settings.CANONICAL_URL + static('img/firefox/new/browser-high-res.png') }}",
+      "{{ settings.CANONICAL_URL + static('img/firefox/home/quantum/hero-laptop-high-res.png') }}",
+      "https://ffp4g1ylyit3jdyti1hqcvtb-wpengine.netdna-ssl.com/press-uk/files/2017/11/fox1.png"
+    ],
+    "name": "Firefox Browser Linux",
+    "alternateName": [
+      "Firefox Linux",
+      "Firefox Browser for Linux"
+    ],
+    "brand":
+    {
+      "@id": "https://www.mozilla.org/#firefoxbrand"
+    },
+    "offers":
+    {
+      "@type": "Offer",
+      "url": "https://www.mozilla.org/en-US/firefox/linux/",
+      "priceCurrency": "USD",
+      "price": "0",
+      "availability": "https://schema.org/InStock"
+    }
+  },
+  {
+    "@type": "Product",
+    "@id": "https://www.mozilla.org/#firefoxlite",
+    "url": "https://support.mozilla.org/en-US/kb/get-started-firefox-lite",
+    "logo": "{{ settings.CANONICAL_URL + static('img/logos/firefox/logo-lite-high-res.png') }}",
+    "image": [
+      "https://lh3.googleusercontent.com/j_OAW-oYnMuRB6FJnAQX9jHTzBy5afe8vyW7BKpXFbu2Gsck-b8W-3KoRTM0_cDkUuU=w1440-h620",
+      "https://lh3.googleusercontent.com/ks-aF3UN4CJahyHt2apAlRtU-O-By9GHKQ-O4SPjpLvejoSkkwi-1Ws1uj_M-A1f4tI=w1440-h620",
+      "https://lh3.googleusercontent.com/mWlOP2E22njyJMkK6gM20qWNe39rqr5nL_U5xt8lzk-Yv4xhMncENoRjNf37hcLbBhU=w1440-h620"
+    ],
+    "name": "Firefox Lite",
+    "brand":
+    {
+      "@id": "https://www.mozilla.org/#firefoxbrand"
+    },
+    "sameAs": [
+      "https://play.google.com/store/apps/details?id=org.mozilla.rocket"
+    ],
+    "offers":
+    {
+      "@type": "Offer",
+      "url": "https://play.google.com/store/apps/details?id=org.mozilla.rocket",
+      "priceCurrency": "USD",
+      "price": "0",
+      "availability": "https://schema.org/InStock"
+    }
+  },
+  {
+    "@type": "Product",
+    "@id": "https://www.mozilla.org/#firefoxlockwise",
+    "url": "https://www.mozilla.org/en-US/firefox/lockwise/",
+    "logo": "{{ settings.CANONICAL_URL + static('protocol/img/logos/firefox/lockwise/logo-lg-high-res.png') }}",
+    "image": [
+      "{{ settings.CANONICAL_URL + static('img/structured-data/lockwise.png') }}",
+      "{{ settings.CANONICAL_URL + static('img/structured-data/lockwise-onboarding.png') }}",
+      "{{ settings.CANONICAL_URL + static('img/structured-data/lockwise-autofill.png') }}"
+    ],
+    "name": "Firefox Lockwise",
+    "brand":
+    {
+      "@id": "https://www.mozilla.org/#firefoxbrand"
+    },
+    "sameAs": [
+      "https://en.wikipedia.org/wiki/Firefox_Lockwise",
+      "https://www.wikidata.org/wiki/Q64829394",
+      "https://play.google.com/store/apps/details?id=mozilla.lockbox",
+      "https://apps.apple.com/us/app/firefox-lockbox/id1314000270"
+    ],
+    "offers":
+    {
+      "@type": "Offer",
+      "url": "https://lockwise.firefox.com",
```

Comment:
https://www.mozilla.org/en-US/firefox/lockwise/ ?

Reply:
I have missed that, thank you